src/twisted/conch/ssh/keys.py

```diff
@@ -1809,6 +1820,22 @@ def verify(self, signature, data):
         else:
             return True
 
+    def isSecurityKey(self):
+        """
+        Return True if key is an OpenSSH security key.
+        """
+        return self.sshType() in [
+            b"sk-ecdsa-sha2-nistp256@openssh.com",
+            b"sk-ssh-ed25519@openssh.com",
+        ]
+
+    def application(self):
```

Comment:
Very minor comment... maybe have it a property.

```suggestion
    @property
    def application(self):
```